.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -312,178 +291,26 @@ jobs:
-    - name: SYCL End-to-end tests
-      shell: bash {0}
+    - name: Run E2E Tests
       if: inputs.tests_selector == 'e2e'
-      env:
-        LIT_OPTS: -v --no-progress-bar --show-unsupported --show-pass --show-xfail --max-time 3600 --time-tests --param test-mode=${{ inputs.e2e_testing_mode }} --param sycl_devices=${{ inputs.target_devices }} ${{ inputs.extra_lit_opts }}
-      run: |
-        ninja -C build-e2e check-sycl-e2e > e2e.log 2>&1
-        exit_code=$?
-        cat e2e.log
-        if [ $exit_code -ne 0 ]; then
-          awk '/^Failed Tests|Unexpectedly Passed Tests|Unresolved tests|Testing Time/{flag=1}/FAILED: CMakeFiles/{flag=0}flag' e2e.log >> $GITHUB_STEP_SUMMARY
-        fi
-        exit $exit_code
-    - name: Build SYCL CTS tests
-      if: inputs.tests_selector == 'cts' && inputs.sycl_cts_artifact == ''
-      env:
-        CMAKE_EXTRA_ARGS: ${{ inputs.extra_cmake_args }}
-      run: |
-        cts_exclude_filter=""
-        # If CTS_TESTS_TO_BUILD is null - use filter
-        if [ -z "$CTS_TESTS_TO_BUILD" ]; then
-          if [ "${{ contains(inputs.cts_testing_mode, 'build-only')  }}" = "true" ]; then
-            cts_exclude_filter=$PWD/devops/cts_exclude_filter_compfails
-          elif [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
-            cts_exclude_filter=$PWD/devops/cts_exclude_filter_OCL_CPU
-          elif [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
-            cts_exclude_filter=$PWD/devops/cts_exclude_filter_L0_GPU
-          fi
-
-          # List excluded SYCL CTS categories:
-          # SYCL_CTS_EXCLUDE_TEST_CATEGORIES - Optional file specifying a list
-          # of test categories to be excluded from the build.
-          echo "::group::Excluded test categories"
-          cat $cts_exclude_filter
-          echo "::endgroup::"
-        fi
-
-        cmake -GNinja -B./build-cts -S./khronos_sycl_cts -DCMAKE_CXX_COMPILER=$(which clang++) \
-        -DSYCL_IMPLEMENTATION=DPCPP \
-        -DSYCL_CTS_EXCLUDE_TEST_CATEGORIES="$cts_exclude_filter" \
-        -DSYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS=OFF \
-        -DDPCPP_INSTALL_DIR="$(dirname $(which clang++))/.." \
-        $CMAKE_EXTRA_ARGS
-        # Ignore errors so that if one category build fails others still have a
-        # chance to finish and be executed at the run stage. Note that
-        # "test_conformance" target skips building "test_all" executable.
-        ninja -C build-cts -k0 $( [ -n "$CTS_TESTS_TO_BUILD" ] && echo "$CTS_TESTS_TO_BUILD" || echo "test_conformance")
-
-    - name: Pack SYCL-CTS binaries
-      if: always() && !cancelled() && inputs.cts_testing_mode == 'build-only'
-      run: tar -I 'zstd -9' -cf sycl_cts_bin.tar.zst -C ./build-cts/bin .
-
-    - name: Upload SYCL-CTS binaries
-      if: always() && !cancelled() && inputs.cts_testing_mode == 'build-only'
-      uses: actions/upload-artifact@v4
+      uses: ./devops/actions/run-tests/e2e
```

Comment:
I tried `uses: ./devops/actions/run-tests/${{inputs.tests_selector}}` but it doesn't work: https://github.com/intel/llvm/actions/runs/12898787585